backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java

```diff
@@ -1,75 +1,41 @@
 package gov.cdc.usds.simplereport.service;
 
-import static gov.cdc.usds.simplereport.api.Translators.parsePersonRole;
-import static gov.cdc.usds.simplereport.api.Translators.parsePhoneType;
-import static gov.cdc.usds.simplereport.api.Translators.parseUserShortDate;
-import static gov.cdc.usds.simplereport.api.Translators.parseYesNo;
-import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.convertEthnicityToDatabaseValue;
-import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.convertRaceToDatabaseValue;
-import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.convertSexToDatabaseValue;
-
-import com.fasterxml.jackson.databind.MappingIterator;
 import gov.cdc.usds.simplereport.api.model.errors.CsvProcessingException;
 import gov.cdc.usds.simplereport.api.model.filerow.PatientUploadRow;
 import gov.cdc.usds.simplereport.api.uploads.PatientBulkUploadResponse;
 import gov.cdc.usds.simplereport.config.AuthorizationConfiguration;
-import gov.cdc.usds.simplereport.db.model.Facility;
-import gov.cdc.usds.simplereport.db.model.Organization;
-import gov.cdc.usds.simplereport.db.model.Person;
-import gov.cdc.usds.simplereport.db.model.PhoneNumber;
-import gov.cdc.usds.simplereport.db.model.auxiliary.StreetAddress;
 import gov.cdc.usds.simplereport.db.model.auxiliary.UploadStatus;
 import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
-import gov.cdc.usds.simplereport.validators.CsvValidatorUtils;
 import gov.cdc.usds.simplereport.validators.FileValidator;
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
 import java.util.UUID;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
 
 /**
  * Service to upload a roster of patient data given a CSV input. Formerly restricted to superusers
- * but now (almost) available to end users.
+ * but now available to end users.
  *
- * <p>Updated by emmastephenson on 10/24/2022
+ * <p>Updated by emmastephenson on 12/2/2022
  */
 @Service
-@Transactional
 @RequiredArgsConstructor
 @Slf4j
 public class PatientBulkUploadService {
 
-  private final PersonService _personService;
-  private final AddressValidationService _addressValidationService;
-  private final OrganizationService _organizationService;
   private final FileValidator<PatientUploadRow> _patientBulkUploadFileValidator;
+  private final PatientBulkUploadServiceAsync _patientBulkUploadServiceAsync;
```

Comment:
the underscore before local variable names violates the [java naming conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Except%20for%20variables%2C%20all%20instance%2C%20class%2C%20and%20class%20constants%20are%20in%20mixed%20case%20with%20a%20lowercase%20first%20letter.%20Internal%20words%20start%20with%20capital%20letters.%20Variable%20names%20should%20not%20start%20with%20underscore%20_%20or%20dollar%20sign%20%24%20characters%2C%20even%20though%20both%20are%20allowed.)

Reply:
But what if I disagree with the style guides? 🤣 

JK, will remove in the subsequent PR. (I am curious why this style fell out of favor though, I really like keeping class-level fields and in-method variables distinguished with the underscore)

Reply:
Yeah, its a convention in c++, its kinda nice, but its not really necessary in java convention since it pushes the getter/setter accessor patterns and you shouldn't have public local variables, because of that you end up underscoring almost everything anyway, which is not really that useful.

Reply:
That's true, I didn't think about the possibility of it encouraging public local variables. Created this ticket to make sure it gets addressed: https://github.com/CDCgov/prime-simplereport/issues/4857

Reply:
And it's even in the wiki now: https://github.com/CDCgov/prime-simplereport/wiki/SimpleReport-Style-Guide#variable-names